1.10/installing/custom/system-requirements/install-docker-centos.md

```diff
@@ -20,6 +20,13 @@ In addition to the general [Docker requirements and recommendations for DC/OS][1
   mkfs -t xfs -n ftype=1 /dev/sdc1
   ```
 
+* If you are using Docker 1.12 or Docker 1.13, you must add `MountFlags=private` in `/etc/systemd/system/docker.service.d/execstart.conf` and then restart the Docker daemon with this command:
```

Comment:
Using MountFlags=private will cause some docker feature not working. See the discussion here:
https://github.com/moby/moby/issues/25718#issuecomment-251785457

Mainly, if you're using or relying on the following docker features, you should not turn on this. If you don't rely on those features, it's recommended to use that setting to avoid possible EBUSY errors.
1) volumes exposed by plugins (containerized volume plugins)
2) volumes exposed by local drivers and mounting a host directory.

I would simply warn the user about this issue and have a link to the docker issue
https://github.com/moby/moby/issues/25718
https://github.com/moby/moby/issues/22260

Which are both unresolved yet.